### DISCUSSION TITLE
Getting "Wrong username or password" on SSO with OIDC (Authentik)

### DISCUSSION THREAD

**Discussion by Dan6erbond** [directus/directus#15879] [Archive]
Hello everyone, I'm trying to configure Directus with Authentik using OIDC, but it's not going as smoothly as I'd like. My config looks like this:

```env
AUTH_PROVIDERS="authentik"

AUTH_AUTHENTIK_DRIVER="openid"
AUTH_AUTHENTIK_CLIENT_ID="..."
AUTH_AUTHENTIK_CLIENT_SECRET="..."
AUTH_AUTHENTIK_SCOPE="openid profile email"
AUTH_AUTHENTIK_ISSUER_URL="https://myauthentik.ch/application/o/strapi-dev/.well-known/openid-configuration"
AUTH_AUTHENTIK_IDENTIFIER_KEY="email"
AUTH_AUTHENTIK_ALLOW_PUBLIC_REGISTRATION=true
AUTH_AUTHENTIK_REQUIRE_VERIFIED_EMAIL=true
AUTH_AUTHENTIK_DEFAULT_ROLE_ID="..."
AUTH_AUTHENTIK_ICON="user-lock"
```

After logging in with Authentik, I get redirected to http://localhost:8055/admin/login?reason=INVALID_CREDENTIALS and see the following message in the UI:

> Wrong username or password

This is my log output:

```sh
08:39:19 Γ£¿ request completed GET 200 /assets/ae486775-a873-4356-a411-0ed163442fb9?download=true 24ms
08:39:21 ≡ƒöì [2.430ms] select "ip_access" from "directus_roles" where "id" is null limit $1 [1]
08:39:21 Γ£¿ request completed GET 302 /auth/login/authentik?redirect=http://localhost:8055/admin/login?continue 14ms
08:39:24 ≡ƒöì [15.795ms] select "ip_access" from "directus_roles" where "id" is null limit $1 [1]
08:39:24 ≡ƒöì RPError [OpenID] Unknown RP error
08:39:25 ΓÜá∩╕Å  Invalid user credentials.
08:39:25 Γ£¿ request completed GET 302 /auth/login/authentik/callback?code=91b9d3dc86364684b4f7473228e8a35b&state=sYy-3XnU8erW2O7AyGUpFooEdipnNPhPd5UyIwn__DE 524ms
```

My Authentik provider config looks like this:

![image](https://user-images.githubusercontent.com/32235074/193997186-20216ed9-cb34-4b87-8bf9-270a0ee1d5a8.png)

![image](https://user-images.githubusercontent.com/32235074/193997240-19483838-28dc-4390-8b80-796631f88709.png)

![image](https://user-images.githubusercontent.com/32235074/193997474-6109d412-c586-4d00-b850-753269435bb8.png)

I saw #15857 and was hoping to find a solution there, but it looks like they were able to get it working so the issue should be somewhere in my config. 

Thank you in advance!

**Comment by azrikahar** [comment]
This might be an issue with authentik as mentioned here with a similar error: https://github.com/directus/directus/issues/15888#issuecomment-1271294406
